package.json

```diff
@@ -35,6 +35,7 @@
     "moment": "^2.24.0",
     "node": "^13.12.0",
     "peerjs": "^1.1.0",
+    "prerender-spa-plugin": "^3.4.0",
```

Comment:
this is very important to prerender the components,.. becuase the pnp is consuming time to load the app!

Reply:
Nice. Worth a try. It probably won't apply to the timeline page, but others that are more static in nature can benefit.